Nym Delegation Program Rules

JohnSmith-PrivacyRebels opened the pull request:
These are proposed Rules to govern DP and keep everything in one place rather than fragmented on various platforms.

JohnSmith-PrivacyRebels committed:
Update isp-sheet.csv

Added a few known VPS providers which (a) support crypto payment (b) allow TOR in some shape or form (c) more or less know for their stability. Will add more eventually.

JohnSmith-PrivacyRebels committed:
Update isp-sheet.csv

added a few more providers

JohnSmith-PrivacyRebels committed:
Add files via upload

This is proposed Rules to govern Nym Delegation Program. Has been discussed with sudo.

JohnSmith-PrivacyRebels committed:
Merge branch 'nymtech:develop' into develop

sudonym2 reviewed (changes requested):
Can we please remove 4/e? It is a duplicate of 4/h.

JohnSmith-PrivacyRebels committed:
Update DP Rules.md

removing 4e as it is a dup

JohnSmith-PrivacyRebels commented:
Done, 4e removed.

JohnSmith-PrivacyRebels committed:
Update DP Rules.md

Added a new rule re 100% saturation (Rule 3) and fixed the formatting of paragraphs slightly.

JohnSmith-PrivacyRebels committed:
Update DP Rules.md

further formatting by adding BR and &nbsp tags

JohnSmith-PrivacyRebels committed:
Update DP Rules.md

Small changes to Welcome sentence, Rule 3 and XX to 48 h

JohnSmith-PrivacyRebels committed:
Update DP Rules.md

Rule 15 de-dup

tommyv1987 commented:
@sudonym2 what do we want to do with this ticket?
